src/Workspaces/Core/Portable/Workspace/Host/Status/WorkspaceStatusService.cs

```diff
@@ -13,6 +14,15 @@ internal sealed class WorkspaceStatusService : IWorkspaceStatusService
     {
         public readonly static WorkspaceStatusService Default = new WorkspaceStatusService();
 
+#pragma warning disable 0067
```

Comment:
:grey_exclamation: There is no need to disable these warnings. Use an explicit interface implementation instead.

Reply:
sure.